docs/getting-started.md

```diff
@@ -109,12 +113,15 @@ To install and run Semgrep OSS Engine, use one of the following options:
       ```
 
   3. Run recommended Semgrep Registry rules:
-    ```bash
-    docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep --config=auto
-    ```
-
-    The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.
-
+      1. On **macOS or Linux**, in the directory to scan:
+        ```bash
+        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep --config=auto
+        ```
+        The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.
+      2. On **WSL**, in the directory to scan:
```

Comment:
@s-santillan, I would say here:
2. On **Windows** ...
as it is the Windows command.

Reply:
@sebasrevuelta Thanks! Yes it is fixed now :)